governance/code-of-conduct.md

```diff
@@ -1,115 +1,6 @@
 # CHAOSS Code of Conduct
 
-## Our Pledge
+This page is a placeholder for the code of conduct page in the governance repo. The actual code of conduct is maintained at the .github level at the following link:
```

Comment:
Maybe something like this, which is more explicit than calling it a placeholder?
```suggestion
Participation in the CHAOSS project is subject to our Code of Conduct, which can be found in our .github repo at
```